modules/blockly/utils/workspace_management/validate_workspace_functions.js

```diff
@@ -143,6 +150,59 @@ module.exports = {
       }
     }
 
+    return true;
+  },
+  /*
+  Function used to check that forms are correctly defined
+  Args : Blockly & formBlock ( the block_slash_command_form_creator block )
+  Will return true if everything correct
+  */
+  checkIfFormCorrectlyDefined: function (Blockly, formBlock) {
+    //This function takes the form block, and will return true if valid, false if not.
+    //Regex already checked in generator
+
+    //Checking if at least one field defined
+    if (formBlock.getInputTargetBlock("INPUTS")) {
+      //There is a block attached here, must be an input
+      if (formBlock.getInputTargetBlock("INPUTS").type !== "block_slash_command_form_input_text") return false;
```

Comment:
Different types of input may be added by Discord.js in the future, so maybe .startsWith is better there. Not a big problem though